war/src/main/scss/modules/side-panel-tasks.scss

```diff
@@ -48,7 +50,7 @@
 }
 
 #tasks .task .task-link {
-  .item();
+  @include mixins.item;
```

Comment:
This adds `position: relative;` and `z-index: 0;` that were not present in the original CSS. Was this intentional?

Reply:
Hm they should have been there originally. I've removed the additions of them from .task-link as they were duplicating what was in mixins.item;